libs/wingsdk/src/shared-aws/function.inflight.ts

```diff
@@ -86,47 +29,80 @@ export class FunctionClient implements IFunctionClient {
       );
     }
     if (!response.Payload) {
-      return { payload: "" };
+      return "";
     }
     const value = JSON.parse(toUtf8(response.Payload)) ?? "";
+    if (typeof value !== "string") {
```

Comment:
Should we fail here or gracefully return `nil` or something?

Reply:
Currently the `invoke` API returns a string, so I don't think returning nil would make sense here. (We need to broaden the supported types for `cloud.Function` to something like `Json` -- but I think it's best to leave that for another PR). Also note that if the user's function returned nothing, the operation above should default to an empty string. Given that, failing fast seems appropriate for now.